src/libraries/Directory.Build.props

```diff
@@ -4,6 +4,17 @@
   </PropertyGroup>
   <Import Project="..\..\Directory.Build.props" />
 
+  <ItemGroup>
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\ref\**\Microsoft.Extensions.*proj" Exclude="@(PortedExtensionsProject)" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\src\**\Microsoft.Extensions.*proj" Exclude="@(PortedExtensionsProject)" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\Microsoft.Extensions.*.Tests.csproj" Exclude="@(PortedExtensionsProject)" />
+  </ItemGroup>
```

Comment:
Nice place to put the central workaround for the traversal projects while you bring them up 👍 👍 👍